phase2/Kconfig

```diff
@@ -39,6 +39,14 @@ config phase2.kubeadm.version
 
 	  Valid options are "stable" or a Google Cloud Storage link to a build.
 
+config phase2.kubeadm.master_upgrade.method
+	string "kubeadm master upgrade method"
+	default "upgrade"
+	help
+	  The kubeadm method of master upgrade to use.
+
+	  Valid options are (upgrade,init).
```

Comment:
Didn't we change this now to `kubeadm-upgrade` and `kubeadm-init` and not scope the upgrade functionality under kubeadm?

Reply:
kubeadm-init is not formally supported. However, we do need this here so we can use it for e2e testing. I can hide it it the comments to make it less confusing what is supported and what is not supported.

Reply:
After looking at the previous code, @luxas's comment, and the new code, I think you might have misinterpreted his request. He can correct me, but I think he was just saying that we should call these "kubeadm-upgrade" and "kubeadm-init" because these methods are specific to kubeadm. Upgrading isn't available for ignition yet (or maybe ever?), but if it were, it might have other methods like "ignition-rocket" and "ignition-swapinplace." Adding the prefix just makes it very clear what the method is used by.

I'm also not sure what you mean by "kubeadm-init is not formally supported." Do you mean within the context of kubernetes-anywhere? Because at least for kubeadm, upgrading clusters via `kubeadm init` is definitely supported, which is the reason we want it e2e tested in the first place.